src/Apis/index.ts

```diff
@@ -231,5 +231,22 @@ export class Apis extends Base {
       body: JSON.stringify(payload)
     });
   }
+
+  executeWorkflow<T extends boolean>(workflow_id:string,slug:string,linked_acc:string,sync_execution:T,payload:any) :Promise<T extends false? {
+    execution_id: string,
+  }: {
+    execution_id: string,
+    return_value?: any
+  }> {
+    return this.request(`/api/v1/workflow/${workflow_id}/execute`, {
+      method : "POST",
+    body: JSON.stringify(payload),
+     headers: {
+        "linked_account_id": linked_acc,
+        "slug": slug,
+        "sync_execution": sync_execution?"true":"false"
```

Comment:
Are they resolved to boolean values at the controller of workflow? Can we send a boolean from here?

Reply:
![image](https://github.com/user-attachments/assets/15f5f1dd-27d3-4c3c-b829-eb5b808b301a)

Yup they are , cause server parse them
![image](https://github.com/user-attachments/assets/922e8406-8d11-445a-8ee4-8836fe3f31f9)

and can't send it from client cause axios headers are explicitly marked to be send as string value
![image](https://github.com/user-attachments/assets/c81c893b-1df4-4bb6-9880-659b5ed4b9fd)